Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 581, in configure
    handler = self.configure_handler(handlers[name])
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 854, in configure_handler
    result = factory(**kwargs)
             ^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/handlers.py", line 214, in __init__
    BaseRotatingHandler.__init__(self, filename, 'a', encoding=encoding,
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/handlers.py", line 58, in __init__
    logging.FileHandler.__init__(self, filename, mode=mode,
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py", line 1231, in __init__
    StreamHandler.__init__(self, self._open())
                                 ^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py", line 1263, in _open
    return open_func(self.baseFilename, self.mode,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/bytedance/code/study/pyStudy/E:/Djlog/file.log'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/bytedance/code/study/pyStudy/pyStudy/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/utils/log.py", line 76, in configure_logging
    logging_config_func(logging_settings)
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 920, in dictConfig
    dictConfigClass(config).configure()
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 588, in configure
    raise ValueError('Unable to configure handler '
ValueError: Unable to configure handler 'file_handler'
unable to load app 0 (mountpoint='') (callable not found or import error)
--- no python application found, check your startup logs for errors ---
[pid: 96509|app: -1|req: -1/1] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 03:40:38 2024] GET /polls/ => generated 21 bytes in 9 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 581, in configure
    handler = self.configure_handler(handlers[name])
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 854, in configure_handler
    result = factory(**kwargs)
             ^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/handlers.py", line 214, in __init__
    BaseRotatingHandler.__init__(self, filename, 'a', encoding=encoding,
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/handlers.py", line 58, in __init__
    logging.FileHandler.__init__(self, filename, mode=mode,
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py", line 1231, in __init__
    StreamHandler.__init__(self, self._open())
                                 ^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py", line 1263, in _open
    return open_func(self.baseFilename, self.mode,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/bytedance/code/study/pyStudy/E:/Djlog/file.log'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/bytedance/code/study/pyStudy/pyStudy/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/utils/log.py", line 76, in configure_logging
    logging_config_func(logging_settings)
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 920, in dictConfig
    dictConfigClass(config).configure()
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 588, in configure
    raise ValueError('Unable to configure handler '
ValueError: Unable to configure handler 'file_handler'
unable to load app 0 (mountpoint='') (callable not found or import error)
--- no python application found, check your startup logs for errors ---
[pid: 96500|app: -1|req: -1/2] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 03:40:39 2024] GET /polls/ => generated 21 bytes in 5 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 581, in configure
    handler = self.configure_handler(handlers[name])
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 854, in configure_handler
    result = factory(**kwargs)
             ^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/handlers.py", line 214, in __init__
    BaseRotatingHandler.__init__(self, filename, 'a', encoding=encoding,
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/handlers.py", line 58, in __init__
    logging.FileHandler.__init__(self, filename, mode=mode,
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py", line 1231, in __init__
    StreamHandler.__init__(self, self._open())
                                 ^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py", line 1263, in _open
    return open_func(self.baseFilename, self.mode,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/bytedance/code/study/pyStudy/E:/Djlog/file.log'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/bytedance/code/study/pyStudy/pyStudy/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/utils/log.py", line 76, in configure_logging
    logging_config_func(logging_settings)
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 920, in dictConfig
    dictConfigClass(config).configure()
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 588, in configure
    raise ValueError('Unable to configure handler '
ValueError: Unable to configure handler 'file_handler'
unable to load app 0 (mountpoint='') (callable not found or import error)
--- no python application found, check your startup logs for errors ---
[pid: 96500|app: -1|req: -1/3] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 03:40:40 2024] GET /polls/ => generated 21 bytes in 5 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 581, in configure
    handler = self.configure_handler(handlers[name])
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 854, in configure_handler
    result = factory(**kwargs)
             ^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/handlers.py", line 214, in __init__
    BaseRotatingHandler.__init__(self, filename, 'a', encoding=encoding,
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/handlers.py", line 58, in __init__
    logging.FileHandler.__init__(self, filename, mode=mode,
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py", line 1231, in __init__
    StreamHandler.__init__(self, self._open())
                                 ^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py", line 1263, in _open
    return open_func(self.baseFilename, self.mode,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/bytedance/code/study/pyStudy/E:/Djlog/file.log'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/bytedance/code/study/pyStudy/pyStudy/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/utils/log.py", line 76, in configure_logging
    logging_config_func(logging_settings)
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 920, in dictConfig
    dictConfigClass(config).configure()
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 588, in configure
    raise ValueError('Unable to configure handler '
ValueError: Unable to configure handler 'file_handler'
unable to load app 0 (mountpoint='') (callable not found or import error)
--- no python application found, check your startup logs for errors ---
[pid: 96500|app: -1|req: -1/4] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 03:40:40 2024] GET /polls/ => generated 21 bytes in 5 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 581, in configure
    handler = self.configure_handler(handlers[name])
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 854, in configure_handler
    result = factory(**kwargs)
             ^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/handlers.py", line 214, in __init__
    BaseRotatingHandler.__init__(self, filename, 'a', encoding=encoding,
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/handlers.py", line 58, in __init__
    logging.FileHandler.__init__(self, filename, mode=mode,
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py", line 1231, in __init__
    StreamHandler.__init__(self, self._open())
                                 ^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py", line 1263, in _open
    return open_func(self.baseFilename, self.mode,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/bytedance/code/study/pyStudy/E:/Djlog/file.log'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/bytedance/code/study/pyStudy/pyStudy/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/utils/log.py", line 76, in configure_logging
    logging_config_func(logging_settings)
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 920, in dictConfig
    dictConfigClass(config).configure()
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 588, in configure
    raise ValueError('Unable to configure handler '
ValueError: Unable to configure handler 'file_handler'
unable to load app 0 (mountpoint='') (callable not found or import error)
--- no python application found, check your startup logs for errors ---
[pid: 96500|app: -1|req: -1/5] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 03:40:40 2024] GET /polls/ => generated 21 bytes in 4 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 581, in configure
    handler = self.configure_handler(handlers[name])
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 854, in configure_handler
    result = factory(**kwargs)
             ^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/handlers.py", line 214, in __init__
    BaseRotatingHandler.__init__(self, filename, 'a', encoding=encoding,
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/handlers.py", line 58, in __init__
    logging.FileHandler.__init__(self, filename, mode=mode,
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py", line 1231, in __init__
    StreamHandler.__init__(self, self._open())
                                 ^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py", line 1263, in _open
    return open_func(self.baseFilename, self.mode,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/bytedance/code/study/pyStudy/E:/Djlog/file.log'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/bytedance/code/study/pyStudy/pyStudy/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/utils/log.py", line 76, in configure_logging
    logging_config_func(logging_settings)
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 920, in dictConfig
    dictConfigClass(config).configure()
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 588, in configure
    raise ValueError('Unable to configure handler '
ValueError: Unable to configure handler 'file_handler'
unable to load app 0 (mountpoint='') (callable not found or import error)
--- no python application found, check your startup logs for errors ---
[pid: 96500|app: -1|req: -1/6] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 03:40:51 2024] GET /polls/ => generated 21 bytes in 4 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 581, in configure
    handler = self.configure_handler(handlers[name])
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 854, in configure_handler
    result = factory(**kwargs)
             ^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/handlers.py", line 214, in __init__
    BaseRotatingHandler.__init__(self, filename, 'a', encoding=encoding,
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/handlers.py", line 58, in __init__
    logging.FileHandler.__init__(self, filename, mode=mode,
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py", line 1231, in __init__
    StreamHandler.__init__(self, self._open())
                                 ^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py", line 1263, in _open
    return open_func(self.baseFilename, self.mode,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/bytedance/code/study/pyStudy/E:/Djlog/file.log'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/bytedance/code/study/pyStudy/pyStudy/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/utils/log.py", line 76, in configure_logging
    logging_config_func(logging_settings)
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 920, in dictConfig
    dictConfigClass(config).configure()
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 588, in configure
    raise ValueError('Unable to configure handler '
ValueError: Unable to configure handler 'file_handler'
unable to load app 0 (mountpoint='') (callable not found or import error)
--- no python application found, check your startup logs for errors ---
[pid: 96500|app: -1|req: -1/7] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 03:40:52 2024] GET /polls/ => generated 21 bytes in 4 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 581, in configure
    handler = self.configure_handler(handlers[name])
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 854, in configure_handler
    result = factory(**kwargs)
             ^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/handlers.py", line 214, in __init__
    BaseRotatingHandler.__init__(self, filename, 'a', encoding=encoding,
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/handlers.py", line 58, in __init__
    logging.FileHandler.__init__(self, filename, mode=mode,
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py", line 1231, in __init__
    StreamHandler.__init__(self, self._open())
                                 ^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py", line 1263, in _open
    return open_func(self.baseFilename, self.mode,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/bytedance/code/study/pyStudy/E:/Djlog/file.log'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/bytedance/code/study/pyStudy/pyStudy/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/utils/log.py", line 76, in configure_logging
    logging_config_func(logging_settings)
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 920, in dictConfig
    dictConfigClass(config).configure()
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 588, in configure
    raise ValueError('Unable to configure handler '
ValueError: Unable to configure handler 'file_handler'
unable to load app 0 (mountpoint='') (callable not found or import error)
--- no python application found, check your startup logs for errors ---
[pid: 96500|app: -1|req: -1/8] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 03:41:02 2024] GET /polls/ => generated 21 bytes in 2 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 581, in configure
    handler = self.configure_handler(handlers[name])
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 854, in configure_handler
    result = factory(**kwargs)
             ^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/handlers.py", line 214, in __init__
    BaseRotatingHandler.__init__(self, filename, 'a', encoding=encoding,
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/handlers.py", line 58, in __init__
    logging.FileHandler.__init__(self, filename, mode=mode,
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py", line 1231, in __init__
    StreamHandler.__init__(self, self._open())
                                 ^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py", line 1263, in _open
    return open_func(self.baseFilename, self.mode,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/bytedance/code/study/pyStudy/E:/Djlog/file.log'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/bytedance/code/study/pyStudy/pyStudy/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/utils/log.py", line 76, in configure_logging
    logging_config_func(logging_settings)
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 920, in dictConfig
    dictConfigClass(config).configure()
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 588, in configure
    raise ValueError('Unable to configure handler '
ValueError: Unable to configure handler 'file_handler'
unable to load app 0 (mountpoint='') (callable not found or import error)
--- no python application found, check your startup logs for errors ---
[pid: 96500|app: -1|req: -1/9] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 03:41:03 2024] GET /polls/ => generated 21 bytes in 5 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 581, in configure
    handler = self.configure_handler(handlers[name])
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 854, in configure_handler
    result = factory(**kwargs)
             ^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/handlers.py", line 214, in __init__
    BaseRotatingHandler.__init__(self, filename, 'a', encoding=encoding,
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/handlers.py", line 58, in __init__
    logging.FileHandler.__init__(self, filename, mode=mode,
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py", line 1231, in __init__
    StreamHandler.__init__(self, self._open())
                                 ^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/__init__.py", line 1263, in _open
    return open_func(self.baseFilename, self.mode,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/bytedance/code/study/pyStudy/E:/Djlog/file.log'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/bytedance/code/study/pyStudy/pyStudy/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/Users/bytedance/path/to/venv/lib/python3.12/site-packages/django/utils/log.py", line 76, in configure_logging
    logging_config_func(logging_settings)
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 920, in dictConfig
    dictConfigClass(config).configure()
  File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/logging/config.py", line 588, in configure
    raise ValueError('Unable to configure handler '
ValueError: Unable to configure handler 'file_handler'
unable to load app 0 (mountpoint='') (callable not found or import error)
--- no python application found, check your startup logs for errors ---
[pid: 96500|app: -1|req: -1/10] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 03:41:03 2024] GET /polls/ => generated 21 bytes in 4 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
SIGINT/SIGTERM received...killing workers...

worker 6 buried after 1 seconds
worker 4 buried after 1 seconds

worker 5 buried after 1 seconds
worker 1 buried after 1 seconds

worker 13 buried after 1 seconds
worker 7 buried after 1 seconds

worker 16 buried after 2 seconds
worker 15 buried after 2 seconds
worker 14 buried after 2 seconds
worker 12 buried after 2 seconds
worker 11 buried after 2 seconds
worker 10 buried after 2 seconds
worker 9 buried after 2 seconds
worker 8 buried after 2 seconds
[deadlock-detector] pid 96500 was holding lock thunder (0x104418000)
worker 3 buried after 2 seconds
worker 2 buried after 2 seconds
goodbye to uWSGI.
VACUUM: pidfile removed.
*** Starting uWSGI 2.0.26 (64bit) on [Sun Jul 21 11:41:12 2024] ***
compiled with version: Apple LLVM 15.0.0 (clang-1500.3.9.4) on 20 July 2024 08:47:57
os: Darwin-23.5.0 Darwin Kernel Version 23.5.0: Wed May  1 20:13:18 PDT 2024; root:xnu-10063.121.3~5/RELEASE_ARM64_T6030
nodename: GVJ7Q3PWH6
machine: arm64
clock source: unix
pcre jit disabled
detected number of CPU cores: 12
current working directory: /Users/bytedance/code/study/pyStudy
writing pidfile to uwsgi.pid
detected binary path: /Users/bytedance/path/to/venv/bin/uwsgi
chdir() to /Users/bytedance/code/study/pyStudy
your processes number limit is 6000
your memory page size is 16384 bytes
detected max file descriptor number: 1048575
lock engine: OSX spinlocks
thunder lock: enabled
uwsgi socket 0 bound to TCP address 127.0.0.1:8001 fd 6
Python version: 3.12.4 (main, Jun  6 2024, 18:26:44) [Clang 15.0.0 (clang-1500.3.9.4)]
Python main interpreter initialized at 0x105255068
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 3505502 bytes (3423 KB) for 32 cores
*** Operational MODE: preforking+threaded ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 97243)
spawned uWSGI worker 1 (pid: 97258, cores: 2)
spawned uWSGI worker 2 (pid: 97259, cores: 2)
spawned uWSGI worker 3 (pid: 97261, cores: 2)
spawned uWSGI worker 4 (pid: 97262, cores: 2)
spawned uWSGI worker 5 (pid: 97263, cores: 2)
spawned uWSGI worker 6 (pid: 97264, cores: 2)
spawned uWSGI worker 7 (pid: 97265, cores: 2)
spawned uWSGI worker 8 (pid: 97266, cores: 2)
spawned uWSGI worker 9 (pid: 97267, cores: 2)
spawned uWSGI worker 10 (pid: 97268, cores: 2)
spawned uWSGI worker 11 (pid: 97269, cores: 2)
spawned uWSGI worker 12 (pid: 97270, cores: 2)
spawned uWSGI worker 13 (pid: 97271, cores: 2)
spawned uWSGI worker 14 (pid: 97272, cores: 2)
spawned uWSGI worker 15 (pid: 97273, cores: 2)
spawned uWSGI worker 16 (pid: 97274, cores: 2)
unable to stat() /Users/bytedance/code/study/pyStudy/logs/touchforlogrotate, events will be triggered as soon as the file is created
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x105255068 pid: 97269 (default app)
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x105255068 pid: 97267 (default app)
mountpoint  already configured. skip.
mountpoint  already configured. skip.
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x105255068 pid: 97268 (default app)
mountpoint  already configured. skip.
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x105255068 pid: 97271 (default app)
mountpoint  already configured. skip.
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x105255068 pid: 97270 (default app)
mountpoint  already configured. skip.
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x105255068 pid: 97262 (default app)
mountpoint  already configured. skip.
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x105255068 pid: 97259 (default app)
mountpoint  already configured. skip.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x105255068 pid: 97266 (default app)
mountpoint  already configured. skip.
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x105255068 pid: 97265 (default app)
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x105255068 pid: 97273 (default app)
mountpoint  already configured. skip.
mountpoint  already configured. skip.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x105255068 pid: 97264 (default app)
mountpoint  already configured. skip.
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x105255068 pid: 97261 (default app)
mountpoint  already configured. skip.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x105255068 pid: 97274 (default app)
mountpoint  already configured. skip.
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x105255068 pid: 97258 (default app)
mountpoint  already configured. skip.
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x105255068 pid: 97263 (default app)
mountpoint  already configured. skip.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x105255068 pid: 97272 (default app)
mountpoint  already configured. skip.
[pid: 97267|app: 0|req: 1/1] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 03:41:14 2024] GET /polls/ => generated 269 bytes in 33 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 0)
[pid: 97264|app: 0|req: 1/2] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 04:02:31 2024] GET /polls/ => generated 269 bytes in 51 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 2/3] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 04:02:32 2024] GET /polls/ => generated 269 bytes in 3 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 3/4] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 04:02:32 2024] GET /polls/ => generated 269 bytes in 2 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 4/5] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 05:08:09 2024] GET /polls/ => generated 269 bytes in 15 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 5/6] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 05:08:10 2024] GET /polls/ => generated 269 bytes in 4 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 6/7] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 05:08:10 2024] GET /polls/ => generated 269 bytes in 2 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 7/8] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 05:08:10 2024] GET /polls/ => generated 269 bytes in 3 msecs (HTTP/1.1 200) 6 headers in 207 bytes (2 switches on core 1)
[pid: 97264|app: 0|req: 8/9] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 05:08:10 2024] GET /polls/ => generated 269 bytes in 1 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 9/10] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 05:13:46 2024] GET /polls/ => generated 269 bytes in 4 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 10/11] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 05:14:16 2024] GET /polls/ => generated 269 bytes in 3 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 11/12] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 05:14:30 2024] GET /polls/ => generated 269 bytes in 3 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 12/13] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 05:15:10 2024] GET /polls/ => generated 269 bytes in 11 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 13/14] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 05:15:44 2024] GET /polls/ => generated 269 bytes in 4 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 14/15] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 05:19:16 2024] GET /polls/ => generated 269 bytes in 5 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 15/16] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 05:19:21 2024] GET /polls/ => generated 269 bytes in 4 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 16/17] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 05:19:52 2024] GET /polls/ => generated 269 bytes in 4 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 17/18] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 05:21:23 2024] GET /polls/ => generated 269 bytes in 5 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 18/19] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 05:25:20 2024] GET /polls/ => generated 269 bytes in 4 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 19/20] 127.0.0.1 () {64 vars in 1409 bytes} [Sun Jul 21 05:25:34 2024] GET /polls/1/ => generated 710 bytes in 16 msecs (HTTP/1.1 200) 8 headers in 357 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 20/21] 127.0.0.1 () {72 vars in 1601 bytes} [Sun Jul 21 05:25:47 2024] POST /polls/1/vote/ => generated 0 bytes in 10 msecs (HTTP/1.1 302) 7 headers in 237 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 21/22] 127.0.0.1 () {66 vars in 1458 bytes} [Sun Jul 21 05:25:47 2024] GET /polls/1/results/ => generated 173 bytes in 6 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 22/23] 127.0.0.1 () {66 vars in 1458 bytes} [Sun Jul 21 05:26:08 2024] GET /polls/1/results/ => generated 173 bytes in 3 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 23/24] 127.0.0.1 () {62 vars in 1356 bytes} [Sun Jul 21 05:26:17 2024] GET /polls/ => generated 269 bytes in 4 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 24/25] 127.0.0.1 () {64 vars in 1387 bytes} [Sun Jul 21 05:27:46 2024] GET /polls/ => generated 269 bytes in 8 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 25/26] 127.0.0.1 () {66 vars in 1409 bytes} [Sun Jul 21 05:28:19 2024] GET /polls/ => generated 269 bytes in 4 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 26/27] 127.0.0.1 () {66 vars in 1409 bytes} [Sun Jul 21 05:28:26 2024] GET /polls/ => generated 269 bytes in 5 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 27/28] 127.0.0.1 () {66 vars in 1409 bytes} [Sun Jul 21 05:28:41 2024] GET /polls/ => generated 269 bytes in 2 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 28/29] 127.0.0.1 () {64 vars in 1309 bytes} [Sun Jul 21 05:28:41 2024] GET /static/polls/style.css => generated 179 bytes in 5 msecs (HTTP/1.1 404) 6 headers in 214 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 29/30] 127.0.0.1 () {66 vars in 1409 bytes} [Sun Jul 21 05:28:53 2024] GET /polls/ => generated 269 bytes in 3 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 30/31] 127.0.0.1 () {64 vars in 1309 bytes} [Sun Jul 21 05:28:53 2024] GET /static/polls/style.css => generated 179 bytes in 0 msecs (HTTP/1.1 404) 6 headers in 214 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 31/32] 127.0.0.1 () {68 vars in 1480 bytes} [Sun Jul 21 05:28:54 2024] GET /polls/1/results/ => generated 173 bytes in 3 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 32/33] 127.0.0.1 () {66 vars in 1437 bytes} [Sun Jul 21 05:28:56 2024] GET /polls/1/ => generated 710 bytes in 4 msecs (HTTP/1.1 200) 8 headers in 357 bytes (2 switches on core 1)
[pid: 97264|app: 0|req: 33/34] 127.0.0.1 () {66 vars in 1409 bytes} [Sun Jul 21 05:28:57 2024] GET /polls/ => generated 269 bytes in 3 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 34/35] 127.0.0.1 () {64 vars in 1309 bytes} [Sun Jul 21 05:28:57 2024] GET /static/polls/style.css => generated 179 bytes in 1 msecs (HTTP/1.1 404) 6 headers in 214 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 35/36] 127.0.0.1 () {66 vars in 1409 bytes} [Sun Jul 21 05:28:58 2024] GET /admin/ => generated 9874 bytes in 27 msecs (HTTP/1.1 200) 10 headers in 470 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 36/37] 127.0.0.1 () {64 vars in 1315 bytes} [Sun Jul 21 05:28:58 2024] GET /static/admin/css/base.css => generated 179 bytes in 3 msecs (HTTP/1.1 404) 6 headers in 214 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 37/38] 127.0.0.1 () {64 vars in 1325 bytes} [Sun Jul 21 05:28:58 2024] GET /static/admin/css/dark_mode.css => generated 179 bytes in 0 msecs (HTTP/1.1 404) 6 headers in 214 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 38/39] 127.0.0.1 () {64 vars in 1329 bytes} [Sun Jul 21 05:28:58 2024] GET /static/admin/css/nav_sidebar.css => generated 179 bytes in 0 msecs (HTTP/1.1 404) 6 headers in 214 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 39/40] 127.0.0.1 () {64 vars in 1325 bytes} [Sun Jul 21 05:28:58 2024] GET /static/admin/css/dashboard.css => generated 179 bytes in 1 msecs (HTTP/1.1 404) 6 headers in 214 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 40/41] 127.0.0.1 () {64 vars in 1327 bytes} [Sun Jul 21 05:28:58 2024] GET /static/admin/css/responsive.css => generated 179 bytes in 1 msecs (HTTP/1.1 404) 6 headers in 214 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 41/42] 127.0.0.1 () {64 vars in 1299 bytes} [Sun Jul 21 05:28:58 2024] GET /static/admin/js/theme.js => generated 179 bytes in 1 msecs (HTTP/1.1 404) 6 headers in 214 bytes (2 switches on core 1)
[pid: 97264|app: 0|req: 42/43] 127.0.0.1 () {64 vars in 1311 bytes} [Sun Jul 21 05:28:58 2024] GET /static/admin/js/nav_sidebar.js => generated 179 bytes in 0 msecs (HTTP/1.1 404) 6 headers in 214 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 43/44] 127.0.0.1 () {66 vars in 1409 bytes} [Sun Jul 21 05:28:59 2024] GET /polls/ => generated 269 bytes in 2 msecs (HTTP/1.1 200) 6 headers in 207 bytes (2 switches on core 1)
[pid: 97258|app: 0|req: 1/45] 127.0.0.1 () {64 vars in 1309 bytes} [Sun Jul 21 05:28:59 2024] GET /static/polls/style.css => generated 179 bytes in 26 msecs (HTTP/1.1 404) 6 headers in 214 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 44/46] 127.0.0.1 () {66 vars in 1409 bytes} [Sun Jul 21 05:29:01 2024] GET /polls/ => generated 269 bytes in 4 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 45/47] 127.0.0.1 () {64 vars in 1309 bytes} [Sun Jul 21 05:29:01 2024] GET /static/polls/style.css => generated 179 bytes in 1 msecs (HTTP/1.1 404) 6 headers in 214 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 46/48] 127.0.0.1 () {66 vars in 1409 bytes} [Sun Jul 21 05:30:08 2024] GET /polls/ => generated 269 bytes in 6 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 47/49] 127.0.0.1 () {66 vars in 1409 bytes} [Sun Jul 21 05:30:41 2024] GET /polls/ => generated 269 bytes in 5 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 48/50] 127.0.0.1 () {66 vars in 1409 bytes} [Sun Jul 21 15:26:00 2024] GET /polls/ => generated 269 bytes in 11 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 49/51] 127.0.0.1 () {66 vars in 1409 bytes} [Sun Jul 21 15:26:01 2024] GET /polls/ => generated 269 bytes in 4 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 50/52] 127.0.0.1 () {66 vars in 1409 bytes} [Sun Jul 21 15:26:01 2024] GET /polls/ => generated 269 bytes in 3 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 51/53] 127.0.0.1 () {66 vars in 1409 bytes} [Sun Jul 21 15:26:01 2024] GET /polls/ => generated 269 bytes in 2 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 52/54] 127.0.0.1 () {66 vars in 1409 bytes} [Mon Jul 22 02:58:57 2024] GET /polls/ => generated 269 bytes in 10 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 53/55] 127.0.0.1 () {66 vars in 1409 bytes} [Mon Jul 22 02:58:58 2024] GET /polls/ => generated 269 bytes in 3 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 54/56] 127.0.0.1 () {66 vars in 1409 bytes} [Mon Jul 22 14:38:18 2024] GET /polls/ => generated 269 bytes in 42 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 55/57] 127.0.0.1 () {60 vars in 1142 bytes} [Mon Jul 29 02:03:48 2024] GET /favicon.ico => generated 179 bytes in 41 msecs (HTTP/1.1 404) 6 headers in 214 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 56/58] 127.0.0.1 () {64 vars in 1249 bytes} [Mon Jul 29 02:03:49 2024] GET /polls/ => generated 338 bytes in 8 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 57/59] 127.0.0.1 () {64 vars in 1249 bytes} [Mon Jul 29 02:03:50 2024] GET /polls/ => generated 338 bytes in 2 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 58/60] 127.0.0.1 () {64 vars in 1249 bytes} [Mon Jul 29 02:03:50 2024] GET /polls/ => generated 338 bytes in 2 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 59/61] 127.0.0.1 () {64 vars in 1249 bytes} [Mon Jul 29 02:03:50 2024] GET /polls/ => generated 338 bytes in 2 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 60/62] 127.0.0.1 () {64 vars in 1271 bytes} [Mon Jul 29 02:03:52 2024] GET /polls/2/ => generated 345 bytes in 3 msecs (HTTP/1.1 200) 8 headers in 357 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 61/63] 127.0.0.1 () {64 vars in 1271 bytes} [Mon Jul 29 02:03:55 2024] GET /polls/4/ => generated 719 bytes in 2 msecs (HTTP/1.1 200) 8 headers in 357 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 62/64] 127.0.0.1 () {72 vars in 1463 bytes} [Mon Jul 29 02:03:56 2024] POST /polls/4/vote/ => generated 0 bytes in 3 msecs (HTTP/1.1 302) 7 headers in 237 bytes (2 switches on core 1)
[pid: 97264|app: 0|req: 63/65] 127.0.0.1 () {66 vars in 1320 bytes} [Mon Jul 29 02:03:56 2024] GET /polls/4/results/ => generated 182 bytes in 2 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 64/66] 127.0.0.1 () {64 vars in 1271 bytes} [Mon Jul 29 02:04:00 2024] GET /polls/3/ => generated 337 bytes in 2 msecs (HTTP/1.1 200) 8 headers in 357 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 65/67] 127.0.0.1 () {72 vars in 1463 bytes} [Mon Jul 29 02:04:02 2024] POST /polls/3/vote/ => generated 393 bytes in 1 msecs (HTTP/1.1 200) 8 headers in 357 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 66/68] 127.0.0.1 () {64 vars in 1249 bytes} [Mon Jul 29 02:04:06 2024] GET /polls/ => generated 338 bytes in 1 msecs (HTTP/1.1 200) 6 headers in 207 bytes (2 switches on core 1)
[pid: 97264|app: 0|req: 67/69] 127.0.0.1 () {62 vars in 1356 bytes} [Sat Aug  3 13:56:43 2024] GET /polls/ => generated 338 bytes in 46 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 68/70] 127.0.0.1 () {64 vars in 1387 bytes} [Sat Aug  3 13:57:14 2024] GET /polls/ => generated 338 bytes in 2 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 69/71] 127.0.0.1 () {64 vars in 1387 bytes} [Sat Aug  3 13:57:15 2024] GET /polls/ => generated 338 bytes in 4 msecs (HTTP/1.1 200) 6 headers in 207 bytes (2 switches on core 1)
[pid: 97264|app: 0|req: 70/72] 127.0.0.1 () {64 vars in 1387 bytes} [Sat Aug  3 13:57:15 2024] GET /polls/ => generated 338 bytes in 7 msecs (HTTP/1.1 200) 6 headers in 207 bytes (2 switches on core 1)
[pid: 97264|app: 0|req: 71/73] 127.0.0.1 () {64 vars in 1387 bytes} [Sat Aug  3 13:57:15 2024] GET /polls/ => generated 338 bytes in 1 msecs (HTTP/1.1 200) 6 headers in 207 bytes (2 switches on core 1)
[pid: 97264|app: 0|req: 72/74] 127.0.0.1 () {64 vars in 1387 bytes} [Sat Aug  3 13:57:24 2024] GET /polls/ => generated 338 bytes in 1 msecs (HTTP/1.1 200) 6 headers in 207 bytes (2 switches on core 1)
[pid: 97264|app: 0|req: 73/75] 127.0.0.1 () {64 vars in 1387 bytes} [Sat Aug  3 13:57:24 2024] GET /polls/ => generated 338 bytes in 1 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 74/76] 127.0.0.1 () {64 vars in 1387 bytes} [Sat Aug  3 13:57:24 2024] GET /polls/ => generated 338 bytes in 2 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 75/77] 127.0.0.1 () {64 vars in 1387 bytes} [Sat Aug  3 13:57:24 2024] GET /polls/ => generated 338 bytes in 2 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 76/78] 127.0.0.1 () {64 vars in 1343 bytes} [Fri Aug  9 07:28:30 2024] GET /polls/ => generated 338 bytes in 72 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 77/79] 127.0.0.1 () {64 vars in 1343 bytes} [Fri Aug  9 07:28:31 2024] GET /polls/ => generated 338 bytes in 2 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 78/80] 127.0.0.1 () {64 vars in 1343 bytes} [Fri Aug  9 07:28:32 2024] GET /polls/ => generated 338 bytes in 1 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 79/81] 127.0.0.1 () {64 vars in 1343 bytes} [Fri Aug  9 07:28:32 2024] GET /polls/ => generated 338 bytes in 56 msecs (HTTP/1.1 200) 6 headers in 207 bytes (1 switches on core 1)
[pid: 97264|app: 0|req: 80/82] 127.0.0.1 () {64 vars in 1343 bytes} [Fri Aug  9 07:28:32 2024] GET /polls/ => generated 338 bytes in 2 msecs (HTTP/1.1 200) 6 headers in 207 bytes (2 switches on core 1)
...brutally killing workers...
worker 11 buried after 1 seconds
worker 10 buried after 1 seconds
